packages/amplify_auth_plugin_interface/lib/src/SignUp/SignUpOptions.dart

```diff
@@ -13,13 +13,12 @@
  * permissions and limitations under the License.
  */
 
-
-
 class SignUpOptions {
```

Comment:
Why not just make this abstract? https://github.com/aws-amplify/amplify-flutter/pull/697/files#diff-634553a25cf511114cd3409bd3324f52178d09bc6467715a52d9f78e8e1e4b8c

And set a default value on Cognito? https://github.com/aws-amplify/amplify-flutter/pull/697/files#diff-67c9b0e8dc3d1ab4a486c06096d4b78d5eaffb25ec1639a42b7f292dde25a6d2R58

Reply:
The only actual changes in the file are lines 19 and 20. The rest are auto formatting.

Reply:
(the change seems reasonable, it just isn't related to the PR)

Reply:
Ah my bad! Sorry I misread the diff.